library/sha256.c

```diff
@@ -35,8 +35,27 @@
 #include "mbedtls/platform.h"
 
 #if defined(__aarch64__)
+
 #  if defined(MBEDTLS_SHA256_USE_A64_CRYPTO_IF_PRESENT) || \
     defined(MBEDTLS_SHA256_USE_A64_CRYPTO_ONLY)
+#    if !defined(__ARM_FEATURE_CRYPTO)
+#      if defined(__clang__)
+#        if __clang_major__ < 18
+/* TODO: Re-consider above after https://reviews.llvm.org/D131064 merged. */
```

Comment:
I think for explanation is required here, specifically a summary of the issue, which should indicate why `__ARM_FEATURE_CRYPTO` works around it (and we don't need the comment on the next line)